README.md

```diff
@@ -132,21 +134,23 @@ KL_AZURE_APP_NAME=kunlun
 ### Analyze the Application you wish to deploy
 
 Change into your project working directory. For our demo we will create a
-new project directory:
+new project directory and fetch the kunlun tool
 
 ```
 mkdir kunlun-test
 cd kunlun-test
+go get github.com/kunlun/kun-lun
+cd go/src/github.com/kun-lun/kunlun/cmd/kl
```

Comment:
Doesn't this depend on where $GOHOME is? that is there is no guarantee that it will be in the current directory. Should it be `cd $GOHOME...`?

Reply:
I left out the $GOHOME business as it's one more setting to keep track of or set. What I was wondering was if it makes sense to create a $KUNLUN_HOME and pushd the working directory for the kunlun (kl) binary/tool. I'll share that with Thomas/team.

Reply:
My thinking was that once the user installs Kunlun, they really have no need for go. Or GO_HOME. They need to be in the Kunlun working directory. Which is where KUNLUN_HOME came from. We don't need another environmental variable as long we make it easy to find the working directory ($HOME/go/src/github.com/kun-lun/kunlun/cmd/kl or $GO_HOME/...) both look clunky.

Reply:
I don't think it is true to say they need to be in Kunlun home. In fact, why are you changing to the directory at all? After the kl tool is installed you can run kl from anywhere. Indeed, since the tool outputs data into your current directory you should be running it from your project directory. 

This is what the original documentation did. These two lines should not be added. They duplicate the instructions in an earlier section (Installing Kunlun from source).

I believe that if you follow those install steps the instructions work in their original format and all that should be left in this PR is the grammar/spelling changes.